Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -403,8 +403,7 @@ void VolumeVerifier::Start()
 
   m_is_tgc = m_volume.GetBlobType() == BlobType::TGC;
   m_is_datel = m_volume.IsDatelDisc();
-  m_is_not_retail = (m_volume.GetVolumeType() == Platform::WiiDisc && !m_volume.HasWiiHashes()) ||
-                    IsDebugSigned();
```

Comment:
Please keep this. Since the dev private key is effectively public, we want to show the message that says we can't verify that dev disc images haven't been tampered with.

Reply:
There is a specific "problem" entry for when the signature fails to verify. (It combines the check for the TMD signature and the ticket signature into one entry, but I'm assuming that's okay?)